src/market_sell.php

```diff
@@ -191,7 +191,7 @@
 			}
 
 			$price = stripslashes($_POST['price']);
-			$price = ceil($price);
+			$price = ceil((float)$price);
```

Comment:
Instead of using (float), verify the type of sending to prevent CROSS Mismatch, like:

"aaa", using float e ceil the system will crash.

To prevent that crash use the **is_numeric($price)** 

If the is_numeric return false, we need throw a error and prevent the continue of the execution.